fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -4092,6 +4098,9 @@ public void replayModifyTableProperty(short opCode, ModifyTablePropertyOperation
                     }
                 } else if (opCode == OperationType.OP_MODIFY_ENABLE_PERSISTENT_INDEX) {
                     olapTable.setEnablePersistentIndex(tableProperty.enablePersistentIndex());
+                    if (olapTable.isCloudNativeTable()) {
+                        olapTable.setPersistentIndexType(TPersistentIndexType.LOCAL);
```

Comment:
Get from `tableProperty`, instead of using hard code `TPersistentIndexType.LOCAL`?

Reply:
So after we have another persistent index type, we not need to change this code again